package.json

```diff
@@ -8,8 +8,8 @@
   },
   "contributors": [
     {
-      "name": "bluefox",
-      "email": "dogafox@gmail.com"
+      "name": "Matthias Kleine",
```

Comment:
Please add youself as contributor ... not replacing, or? ;-)

Reply:
Bluefox is already author of the package.

Reply:
ok, fair enough :-)